R/checkFormulas.R

```diff
@@ -120,9 +120,9 @@ checkFormulas <- function(d, sheet) {
         "WARNING! In tab ",
         sheet,
         ", ", NROW(cols_affected), " ALTERED FORMULAS:",
-        " Altering formulas without DUIT and PPM approval may lead to programmatic",
-        " and technical issues in your Data Pack.",
-        " Note that this may be due to a formula being deleted",
+        " Altering formulas in the Grey colored columns without DUIT and PPM approval may lead to programmatic",
+        " and technical issues in your Data Pack. Green colored columns however may be altered without permission. ",
+        " Note that this warning  may be triggered due to a formula being deleted",
```

Comment:
Should remove this extra space
```suggestion
        " Note that this warning may be triggered due to a formula being deleted",
```